admin-dev/themes/default/template/controllers/cart_rules/form.js

```diff
@@ -482,7 +482,7 @@ $('.uncombinable_search_filter').typeWatch({
 
 
 $('.combinable_search_filter').typeWatch({
-	captureLength: -1,
+	captureLength: 0,
```

Comment:
captureLength must be -1, if not, content is not reload by ajax if field content is empty